src/Microsoft.AspNetCore.Razor.Language/RazorExtension.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public abstract class RazorExtension
+    {
+        public abstract string ExtensionName { get; }
```

Comment:
I imagine this would eventually have a way to configure a passed in RazorProjectEngine yes?

Reply:
perhaps